examples/Example/src/showcase/betterSwipeable/AppleStyleSwipeableRow.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { useRef } from 'react';
+import { Alert, I18nManager, StyleSheet, Text } from 'react-native';
+import {
+  BetterSwipeable,
+  SwipeableController,
+  RectButton,
+} from 'react-native-gesture-handler';
+import Animated, {
+  Extrapolate,
+  useAnimatedStyle,
+} from 'react-native-reanimated';
+
+export const AppleStyleSwipeableRow: React.FunctionComponent<unknown> = (
+  props
+) => {
+  const controller = useRef<SwipeableController>(null);
+
+  function LeftActions(
+    _progress: Animated.SharedValue<number>,
+    drag: Animated.SharedValue<number>
+  ) {
+    const animatedStyle = useAnimatedStyle(() => {
+      return {
+        transform: [
+          {
+            translateX: Animated.interpolate(
+              drag.value,
+              [0, 50, 100, 101],
+              [-20, 0, 0, 1],
+              Extrapolate.CLAMP
+            ),
+          },
+        ],
+      };
+    });
+
+    return (
+      <RectButton
+        style={styles.leftAction}
+        onPress={controller?.current?.close}>
+        <Animated.View style={animatedStyle}>
+          <Text style={styles.actionText}>Archive</Text>
+        </Animated.View>
+      </RectButton>
+    );
+  }
+
+  function RenderRightAction(
+    text: string,
+    color: string,
+    x: number,
+    progress: Animated.SharedValue<number>
+  ) {
+    const animatedStyles = useAnimatedStyle(() => {
+      return {
+        transform: [
+          { translateX: Animated.interpolate(progress.value, [0, 1], [x, 0]) },
+        ],
+      };
+    });
+
+    return (
+      <Animated.View style={[animatedStyles, { flex: 1 }]}>
+        <RectButton
+          style={[styles.rightAction, { backgroundColor: color }]}
+          onPress={() => {
+            Alert.alert(text);
+            controller?.current?.close();
+          }}>
+          <Text style={styles.actionText}>{text}</Text>
+        </RectButton>
+      </Animated.View>
+    );
+  }
+
+  function RightActions(
+    progress: Animated.SharedValue<number>,
+    _drag: Animated.SharedValue<number>
```

Comment:
```suggestion
    drag: Animated.SharedValue<number>
```

Reply:
What is the reason behind this? Those variables are not used in the functions.